esmf_regrid/experimental/io.py

```diff
@@ -52,11 +52,18 @@ def save_regridder(rg, filename):
     regridder_type = rg.__class__.__name__
     if regridder_type == "GridToMeshESMFRegridder":
         src_grid = (rg.grid_y, rg.grid_x)
-        src_shape = [len(coord.points) for coord in src_grid]
+        if len(src_grid[0].shape) == 1:
```

Comment:
Can we assume that one of these being 1D means that the other is? I guess so, but don't know this code well enough to be sure that this contract will always hold.

Reply:
This should be enforced here: https://github.com/SciTools-incubator/iris-esmf-regrid/blob/266e9ed38588106dec1acc664f2c2fa8bbf476e1/esmf_regrid/schemes.py#L35